services/backend/src/core/profile/profile.js

```diff
@@ -27,6 +27,10 @@ function idHelper(id, savedId) {
   };
 }
 
+function capitalizeFirstLetter(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
```

Comment:
Should we just use lodash here since its already imported? _.upperFirst

Reply:
Ohh didn't know there was a lodash function for that... will make the change